students/jyeon/user/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import UserSignupView
+
+urlpatterns = [
+    path('/usersignup', UserSignupView.as_view())
```

Comment:
signup 정도면 충분히 유저 회원가입이라는 의미를 전달할 수 있으니 가독성이 좋게 signup 으로만 써주셔도 좋습니다.
```suggestion
    path('/signup', UserSignupView.as_view())
```